wrappers/react/src/hotColumn.tsx

```diff
@@ -105,4 +119,9 @@ class HotColumn extends React.Component<HotColumnProps, {}> {
   }
 }
 
-export { HotColumn };
+const HotColumn: React.FC<HotColumnProps> = (props) => {
```

Comment:
this split into two components is only temporary; in step 4 we'll be replacing `HotColumn` class with a function and it will be merged into one component that uses both `HotTableContext` and `HotColumnContext` (something that class-based components can't do easily).

Reply:
That makes sense, thanks for letting me know!